makefile

```diff
@@ -133,21 +152,36 @@ y      := $(notdir $(filter $(_OS)/%,lnx/so win/dll mac/dylib))
 -eGRP  = $(if $(OS_is_lnx),-Wl$(comma)--end-group,)
 daalmake = make
 
+ifeq ($(ARCH_is_arm),yes)
+a8sve_OPT := $(a8sve_OPT.$(COMPILER))
+else
 p4_OPT   := $(p4_OPT.$(COMPILER))
 mc3_OPT  := $(mc3_OPT.$(COMPILER))
 avx2_OPT := $(avx2_OPT.$(COMPILER))
 skx_OPT  := $(skx_OPT.$(COMPILER))
+endif
 
 _OSr := $(if $(OS_is_win),win,$(if $(OS_is_lnx),lin,))
 
+ifeq ($(ARCH_is_arm),yes)
```

Comment:
Hi @keeranroth thanks for suggestion
I feel code is already conditioned as per the compilation flow. If you think any branch can be merged, please suggest we will do it. (Since `makefile` is deprecating in future, does it require this effort?)